src/main/kotlin/calculator/presentation/InputView.kt

```diff
@@ -0,0 +1,7 @@
+package calculator.presentation
+
+class InputView {
```

Comment:
똑같은 기능의 객체를 매번 생성하기보다는 객체 하나를 재사용하는 것은 어떨까요?

```suggestion
object InputView {
```